<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/newPrefix/image.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>191</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Google_radioButton">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Google</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Microsoft_radioButton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Microsoft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Tesla_radioButton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Tesla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Twitter_radioButton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Twitter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Facebook_radioButton">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Facebook</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Ubuntu Condensed&quot;;
border-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Stock Market Prediction System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>551</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="opencsv_pushButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Open csv output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plotdata_pushButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Plot data</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
